### DISCUSSION TITLE
как узнать где произошла ошибка в коде режима

### DISCUSSION THREAD

**Discussion by yurkapotam** [Sokol-Games-GmbH/PixelCombats.PublicApi#133] [Q&A] [answered]
как узнать где произошла ошибка в коде режима

**Comment by ZhenyaMartynov** [comment]
Эх...

**Reply by kkohno** [reply]
ну если есть идея получше - предлагайте. А вообще позже  займемся проработкой дебаггера...

**Comment by kkohno** [comment, marked as answer]
попробуйте логами?

**Comment by justqstn** [comment]
почитай про try catch в js
конструкция 
try {
// код режима
} catch(e) { Validate.ReportInvalid(e.name + "\n" + e.message); }

если ошибка синтаксическая, то не поможет
